NEWS.md

```diff
@@ -10,10 +10,15 @@
   is not specified (#2117, @salim-b).
 
 * usethis's criteria for recognizing a project have expanded to include (#2133):
-  - a `.vscode/` directory, which Positron or VS Code might create
+  - a `.vscode/settings.json` file, which Positron or VS Code might create
   - a `_quarto.yml` file, typical of a Quarto project
   - an `renv.lock` file, which renv creates
 
+* `create_quarto_project()` is a new experimental function that combines basic
+  usage of `quarto::quarto_create_project()` with some of the niceties of
```

Comment:
I think putting `quarto::quarto_create_project()` inside the `usethis::create_*()` envelope is the main contribution here. E.g. challenging a project-within-a-project and opening / switching to the newly created Quarto project.